services/blockchain-indexer/shared/indexer/transactionProcessor/token/transfer.js

```diff
@@ -46,6 +54,15 @@ const applyTransaction = async (blockHeader, tx, dbTrx) => {
 
 	await updateAddressBalanceQueue.add({ address: tx.recipientAddress });
 
+	const account = {
+		address: tx.senderAddress,
+		publicKey: tx.senderPublicKey,
```

Comment:
Don't we only receive `senderPublicKey` within the transaction? `senderAddress` should be computed from the `senderPublicKey`?